recipes/doebase/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "doebase" %}
+{% set version = "v2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pablocarb/doebase/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4c2be23c261dbf47c38d7c1097c8e30be108369a5a2c3f0c6ef6062dc9e06172
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install -vv .
+  skip: true  # [win or py != 37]
```

Comment:
Are you sure you only want this pac kage to work for python 3.7?

Reply:
Hi @beckermr ,
yes this package is only working with python 3.7, thank you.